src/js/ui/window.js

```diff
@@ -116,6 +116,7 @@ Window.prototype._show = function(pushing) {
 
 Window.prototype.show = function() {
   WindowStack.push(this);
+  simply.impl.markDirty();
```

Comment:
with pebble.js we are transmitting the window components to the watch then calling the render, however when we don't need to transmit we need to know when the screen is being shown so we only call render once(ish)

I hope there is a better way to do this with out modifying this file